oauth2_http/java/com/google/auth/oauth2/UserAuthorizer.java

```diff
@@ -161,6 +170,15 @@ public TokenStore getTokenStore() {
     return tokenStore;
   }
 
+  /**
+   * Returns the client authentication type in RFC7591
+   *
+   * @return The ClientAuthenticationType
+   */
+  public ClientAuthenticationType getClientAuthenticationType() {
```

Comment:
Returns the client authentication type as defined in RFC 7591.

@return The {@link ClientAuthenticationType}.